addons/sys_sem70/radio/fnc_openGui.sqf

```diff
@@ -56,15 +56,15 @@ TRACE_2("OpenGui",GVAR(currentRadioId),GVAR(lastAction));
 // Use this to turn off the backlight display//also to save last channel
 
 [{
-    params ["_input","_pfhID"];
+    params ["_input", "_pfhID"];
 
-    if (GVAR(currentRadioId) isEqualTo -1) then {_input set [1,false]}; // Remove PFH on exit.
-    _input params ["_radioId","_open"];
-    if (_open) then { _input set [2,GVAR(lastAction)]; };
+    if (GVAR(currentRadioId) isEqualTo -1) then {_input set [1, false]}; // Remove PFH on exit.
+    _input params ["_radioId", "_open"];
+    if (_open) then { _input set [2, GVAR(lastAction)]; };
     private _lastAction = _input select 2;
 
-    if (_lastAction+5 < time) then {
-        if (GVAR(backlightOn) && !GVAR(displayButtonPressed)) then {
+    if ((_lastAction + 5) < time) then {
```

Comment:
```suggestion
    if (_lastAction + 5 < time) then {
```
Isn't this more readable and how you would write it in C/C++?

Reply:
In C++ this would be written in brackets (MISRA standard)